app/src/components/attachments/FileUpload.test.tsx

```diff
@@ -0,0 +1,108 @@
+import { fireEvent, render, waitFor } from '@testing-library/react';
+import { APIError } from 'hooks/api/useAxios';
+import { useBiohubApi } from 'hooks/useBioHubApi';
+import React from 'react';
+import FileUpload from './FileUpload';
+
+jest.mock('../../hooks/useBioHubApi');
+const mockUseBiohubApi = {
+  project: {
+    uploadProjectArtifacts: jest.fn<Promise<any>, []>()
+  }
+};
+
+const mockBiohubApi = ((useBiohubApi as unknown) as jest.Mock<typeof mockUseBiohubApi>).mockReturnValue(
+  mockUseBiohubApi
+);
+
+const projectId = 1;
+
+const renderContainer = () => {
+  return render(<FileUpload projectId={projectId} />);
+};
+
+describe('FileUpload', () => {
+  it('matches the snapshot', () => {
```

Comment:
can we add a snapshot test for when there is at least one file present in the fileupload component? :)

Reply:
added